view/theme/frio/templates/head.tpl

```diff
@@ -55,6 +55,8 @@
 	media="screen" />
 <link rel="stylesheet" href="view/theme/frio/css/font-awesome.custom.css?v={{$smarty.const.FRIENDICA_VERSION}}"
 	type="text/css" media="screen" />
+<link rel="stylesheet" href="/vendor/enyo/dropzone/dist/min/dropzone.min.css?v={{$smarty.const.FRIENDICA_VERSION}}"
```

Comment:
```suggestion
<link rel="stylesheet" href="vendor/enyo/dropzone/dist/min/dropzone.min.css?v={{$smarty.const.FRIENDICA_VERSION}}"
```

Reply:
removed all occurances of dropzone.min.[js|css] except in view/theme/frio/templates/head.tpl, because this is enough for functionality in all parts.

Reply:
This also removed the Browser error that Dropzone already exists in /compose